Assets/MixedRealityToolkit.Examples/Demos/Gltf/Models/Lantern/README.md

```diff
@@ -0,0 +1,10 @@
+# Lantern
+
+## Screenshot
+
+![screenshot](screenshot/screenshot.jpg)
+
+## License Information
```

Comment:
Same question re: the boombox model.

Reply:
What motivations are there for not using this?

Reply:
Will follow up with other comment thread